docs/rst/notes/galactic/notes.rst

```diff
@@ -1,6 +1,55 @@
 .. _notes_galactic_latest:
 
-Galactic Gamble (v1.0.0)
+Galactic Gamble
+===============
+
+Galactic Gamble (v1.0.2)
 ------------------------
 
-.. Some release notes
+The following packages have been updated:
```

Comment:
In general...
I'd rephrase this sentence as it seems that all packages have been updated.